cmd/cloud_sql_proxy/cloud_sql_proxy.go

```diff
@@ -587,6 +591,12 @@ func main() {
 		RefreshCfgBuffer:   refreshCfgBuffer,
 	}
 
+	var hc *healthcheck.HealthCheck
+
+	if *useHealthCheck {
+		hc = healthcheck.InitHealthCheck(proxyClient)
```

Comment:
Since this function is initializing an object, it's conventional to use `NewHealthCheck`.